lib/file.js

```diff
@@ -356,11 +365,13 @@ class SourceFile extends File {
      */
     getImports() {
         const buffer = new Buffer()
+        buffer.add('import cds from \'@sap/cds\'') // TODO should go to visitor#printService, but can't express this as Path
```

Comment:
There is in fact [one other place](https://github.com/cap-js/cds-typer/blob/main/lib/resolution/builtin.js#L10) where we refer to types from another lib. To circumvent top level imports I had resorted to inline-imports until now. But maybe this is a good occasion to facilitate top level imports in generated files. I am fine with either way, just adding this for completeness sake.

Reply:
Let's leave it like that for now, to not make this PR grow too much.